src/utils/fetch-graphql.js

```diff
@@ -260,33 +280,36 @@ const handleFetchErrors = async ({
         }
          \n${genericError({ url })}`,
         { useVerboseStyle: true }
-      )
+      ),
+      CODES.Authentication
     )
   }
 
   const forbidden = e.message.includes(`Request failed with status code 403`)
 
   if (forbidden) {
-    reporter.panic(
+    structuredReporter.panic(
       formatLogMessage(
         `${e.message}\n\nThe GraphQL request was forbidden.\nIf you are using a security plugin like WordFence or a server firewall you may need to whitelist your IP address or adjust your firewall settings for your GraphQL endpoint.\n\n${errorContext}`
-      )
+      ),
+      CODES.Permissions
```

Comment:
I think this code might be confusing in the context of GraphQL errors since GraphQL requests can have data level permissions. I wonder if this would make more sense as `RequestPermissions` or `RequestDenied`